pkg/kubectl/describe.go

```diff
@@ -2068,3 +2067,28 @@ func (fn typeFunc) Describe(exact interface{}, extra ...interface{}) (string, er
 	}
 	return s, err
 }
+
+// printLabelsMultiline prints multiple labels with a proper alignment.
+func printLabelsMultiline(out io.Writer, title string, labels map[string]string) {
```

Comment:
It's possible this could be called from a context where nesting is necessary - it would probably be better to take the indentation as an argument and pass "\t" down so that we can use it in nested printers.